cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(required=False)
```

Comment:
if you want to make the description not a required field you need to specify this in models as well

Reply:
I chose to make it required 😅